lua/kubectl/utils/tables.lua

```diff
@@ -3,6 +3,7 @@ local hl = require("kubectl.actions.highlight")
 local state = require("kubectl.state")
 local time = require("kubectl.utils.time")
 local M = {}
+local context_rows = {}
```

Comment:
I don't think this is supposed to be a global (for the module) variable?

Reply:
It was being accessed by `addContextRows` as a global variable, but it's been refactored in #440 now anyway 👍